src/common.rs

```diff
@@ -149,3 +187,44 @@ impl Interval {
         }
     }
 }
+
+
+#[derive(Debug)]
+pub struct VariantBuffer {
+    pub buffer: bcf::buffer::RecordBuffer,
```

Comment:
Instead of making these public, I would prefer to move the functionality for update and access into the impl of this struct.